crates/dids/src/document.rs

```diff
@@ -63,9 +65,10 @@ impl KeyIdFragment {
 pub struct Service {
     pub id: String,
     #[serde(rename = "type")]
-    pub r#type: String,
+    pub r#type: OneOrMany<String>,
+    // TODO: Update to allow service endpoint as a map?
     #[serde(rename = "serviceEndpoint")]
-    pub service_endpoint: String,
+    pub service_endpoint: OneOrMany<String>,
```

Comment:
Sorry re-reading this and I don't think I was clear on this bit:

> And it wouldn't help us with binding because even though we constrain, we still have to support 1 multi-type in [the VC's issuer](https://github.com/TBD54566975/web5-spec/blob/main/spec/vc.md#verifiable-credential-data-model)

What I intended to convey is, we could resort back to only implementing `serviceEndpoints` as an array of strings per our `web5-spec`, but that wouldn't resolve the UDL concern because ultimately we still have one multi-type property with the VC `issuer`.

Reply:
> In case you weren't aware we do constrain our set of types [in web5-spec](https://github.com/TBD54566975/web5-spec/blob/main/spec/did.md#service-data-model)

I was not aware, and I'm very glad you flagged that! I'm gonna update to use Vec<String> rather than OneOrMany<String>. Any opportunity for simplicity.

> that wouldn't resolve the UDL concern because ultimately we still have one multi-type property with the VC issuer.

Gotcha. I'm still in favor using Vec<String> regardless, but this is good context.